### PR TITLE
Update user config for descriptor_name for TBS

### PR DESCRIPTION
**Pull request by tylerphelan**
- Also add context for what descriptors are

Which other branches should this be merged with (if any)?

This should only be merged into 1.1

### CODE REVIEW

File: tanzu-build-service/tbs-about.md [hunk 1]
```diff
@@ -1,3 +1,90 @@
...
+
+On non-airgapped clusters, dependencies are imported as a part of installation of a TAP profile or the TBS component.
+
+When creating the values file during installation, include the following bolded keys (in addition to any other `buildservice` keys):
+
+<pre>
+...
+kp_default_repository: REPOSITORY
+kp_default_repository_username: REGISTRY-USERNAME
+kp_default_repository_password: REGISTRY-PASSWORD
```
Comment (tomkennedy513) [inline tanzu-build-service/tbs-about.md:19]: Do we want to include secret ref

**Review by tomkennedy513 — APPROVED** [review]

**Comment by mgibson1121** [comment]
I would also mention that we default installs to using `lite` unless overridden by specifying `full`.

File: tanzu-build-service/tbs-about.md [hunk 2]
```diff
@@ -1,3 +1,90 @@
...
+A full table of differences between `full` and `lite` shown [here](#descriptors-table).
+
+#### <a id="dependencies-descriptors-compared"> Descriptors Compared
+
+Both `lite` and `full` descriptors are suitable for production environments.
+
+|  | lite | full |
+|---|---|---|
+| Faster installation time | Yes | No |
+| Dependencies pre-packaged | No | Yes |
```
[the remark this answers is not in the document]
Comment (tylerphelan) [reply tanzu-build-service/tbs-about.md:72]: @mgibson1121 
I was thinking about this and the deps updater isn't going to work with an fully-airgapped env anyways 😂

So I removed any mention of air-gapped. If users want a non-airgapped install with offline builds then I can add all that context just lmk

File: tanzu-build-service/tbs-about.md (hunk 1 above)
Comment (tylerphelan) [reply tanzu-build-service/tbs-about.md:19]: maybe in a different PR

**Comment by tylerphelan** [comment]
@mgibson1121 
> I would also mention that we default installs to using `lite` unless overridden by specifying `full`.

These are how I tried to address this concern. Happy to add more context.
https://github.com/pivotal/docs-tap/pull/990/files#diff-30db67ca745899dab3d755f0b874ac1ea71401d9169674462dc2abfb8cec4844R41
https://github.com/pivotal/docs-tap/pull/990/files#diff-f8e5611542bab45479efd2bba09bad332b36b8e4abaa627a7e83b1c8d0abbcd6R226

**Comment by tylerphelan** [comment]
ready for merge!

File: install.md [hunk 3]
```diff
@@ -221,9 +221,9 @@ Where:
 - `KP-DEFAULT-REPO-USERNAME` is the username that can write to `KP-DEFAULT-REPO`. You should be able to `docker push` to this location with this credential.
     * For Google Cloud Registry, use `kp_default_repository_username: _json_key`
 - `KP-DEFAULT-REPO-PASSWORD` is the password for the user that can write to `KP-DEFAULT-REPO`. You can `docker push` to this location with this credential.
-    * For Google Cloud Registry, use the contents of the service account JSON key.
-- `DESCRIPTOR-NAME` is the name of the descriptor to import automatically. Current available options at time of release:
-    * `full` contains all dependencies, and is for production use.
+- `DESCRIPTOR-NAME` is the name of the descriptor to import. See more details [here](tanzu-build-service/tbs-about.html#dependencies-descriptors). Available options are:
```
Comment (matthewmcnew) [inline install.md:224]: This is really good.

**Review by matthewmcnew — APPROVED** [review]

**Comment by tylerphelan** [comment]
This PR includes all necessary changes for TBS in TAP 1.1, it's all included in one PR for simplicity

File: install.md [hunk 4]
```diff
@@ -167,10 +167,16 @@ profile: full
...
-  tanzunet_username: "TANZUNET-USERNAME"
-  tanzunet_password: "TANZUNET-PASSWORD"
+  kp_default_repository_username: "KP-DEFAULT-REPO-USERNAME"    # Mutually exclusive with kp_default_repository_secret
+  kp_default_repository_password: "KP-DEFAULT-REPO-PASSWORD"    # Mutually exclusive with kp_default_repository_secret
+  kp_default_repository_secret:                                 # Mutually exclusive with kp_default_repository_username/kp_default_repository_password
+    name: "KP-DEFAULT-REPO-SECRET-NAME"
+    namespace: "KP-DEFAULT-REPO-SECRET-NAMESPACE"
+  tanzunet_username: "TANZUNET-USERNAME"                        # Mutually exclusive with tanzunet_secret
+  tanzunet_password: "TANZUNET-PASSWORD"                        # Mutually exclusive with tanzunet_secret
+  tanzunet_secret:                                              # Mutually exclusive with tanzunet_username/tanzunet_password
```
Comment (matthewmcnew) [inline install.md:177]: I wonder if we should simplify the values by only providing inline secrets or inline secret refs to make copy and pasting as productive as possible.

Comment (matthewmcnew) [reply install.md:177]: Probably only show inline secrets because the steps above do not instruct for the kp-repo-secret to be created.

Comment (tylerphelan) [reply install.md]: @matthewmcnew How about this approach?

**Review by mgibson1121 — APPROVED** [review]

**Comment by tylerphelan** [comment]
Adding more context about managing dependencies

**Comment by tylerphelan** [comment]
Ready!

**Comment by RichardJJG** [comment]
CC @piachakra because affects an installation topic.

**Pull request merged**